src/demos/input-time-picker.html

```diff
@@ -39,103 +40,67 @@
-          await customElements.whenDefined("calcite-tile-select");
-          const inputTimePicker = document.querySelector("calcite-input-time-picker");
-          await inputTimePicker.componentOnReady();
-          const tileSelectGroup = document.querySelector("calcite-tile-select-group");
-          await tileSelectGroup.componentOnReady();
-          const inputTimePickerEls = Array.from(document.querySelectorAll("calcite-input-time-picker"));
-          const timePickerEls = Array.from(document.querySelectorAll("calcite-time-picker"));
+
           document.addEventListener("calciteInputTimePickerChange", (event) => {
             console.log("calciteInputTimePickerChange", event.target.value);
           });
-          document.addEventListener("calciteTileSelectChange", (event) => {
-            const locale = event.target.value;
-            inputTimePickerEls.forEach((inputTimePickerEl) => (inputTimePickerEl.lang = locale));
-            timePickerEls.forEach((timePickerEl) => (timePickerEl.lang = locale));
-          });
-          locales.forEach(({ locale, name }) => {
-            let localeEl = document.createElement("calcite-tile-select");
-            localeEl.setAttribute("heading", name);
-            localeEl.setAttribute("name", "locales");
-            localeEl.setAttribute("scale", "s");
-            localeEl.setAttribute("width", "full");
-            localeEl.setAttribute("value", locale);
-            tileSelectGroup.appendChild(localeEl);
-          });
-          const htmlEl = document.querySelector("html");
-          const htmlElObserver = new MutationObserver((mutations) => {
-            mutations.forEach((mutation) => {
-              if (mutation.type == "attributes" && mutation.attributeName === "dir") {
-                inputTimePickerEls.forEach((inputTimePickerEl) =>
-                  inputTimePickerEl.setAttribute("dir", htmlEl.getAttribute("dir"))
-                );
-                timePickerEls.forEach((timePickerEl) => timePickerEl.setAttribute("dir", htmlEl.getAttribute("dir")));
-              }
-            });
-          });
-          htmlElObserver.observe(htmlEl, {
-            attributes: true
+
+          const mainEl = document.querySelector("main");
+          const currentDate = new Date();
+          const currentTime = `${currentDate.getHours()}:${currentDate.getMinutes()}:${currentDate.getSeconds()}`;
+
+          locales.forEach(({ locale, name, numberingSystem, hourCycle }) => {
+            let labelEl = document.createElement("calcite-label");
+            let inputTimePickerEl = document.createElement("calcite-input-time-picker");
+
+            inputTimePickerEl.setAttribute("lang", locale);
+            if (numberingSystem) {
+              inputTimePickerEl.setAttribute("numbering-system", numberingSystem);
+            }
+            inputTimePickerEl.setAttribute("value", "10:00:00");
+            labelEl.appendChild(document.createTextNode(`${name} (${locale})`));
```

Comment:
Nitpick: Unless truly necessary, let's use `append` vs `appendChild` whenever we can for conciseness.

Reply:
Wow, learn something new about the web API every day!  Wasn't even aware this existed :)